source/Irrlicht/COpenGLExtensionHandler.cpp

```diff
@@ -983,8 +917,7 @@ void COpenGLExtensionHandler::initExtensions(bool stencilBuffer)
 
     num=0;
     glGetIntegerv(GL_MAX_COMBINED_TEXTURE_IMAGE_UNITS, &num);
-    //can we get rid of that? I don't really see reason for this clamp at this level of abstraction
-	MaxTextureUnits = core::min_(static_cast<uint8_t>(num), static_cast<uint8_t>(MATERIAL_MAX_TEXTURES));
+	MaxTextureUnits = num;
```

Comment:
killed the `MATERIAL_MAX_TEXTURES` variable?

Reply:
yea